esmvaltool/diag_scripts/weathertyping/weathertyping.py

```diff
@@ -0,0 +1,222 @@
+from wt_utils import (get_cfg_vars, load_wt_preprocessors,
+                      wt_algorithm, get_ancestors_era5_eobs, calc_slwt_obs,
+                      run_predefined_slwt, combine_wt_to_file, load_wt_files,
+                      get_looping_dict, plot_means, get_model_output_filepath,
+                      calc_lwt_slwt_model, write_lwt_to_file, calc_lwt_model,
+                      plot_seasonal_occurrence
+                      )
+
+import iris
+
+# import internal esmvaltool modules here
+from esmvaltool.diag_scripts.shared import (
+    run_diagnostic,
+)
+
```

Comment:
if you could please run `isort` on this, that'll fix the import order; also - are you importing everything from `wt_utils` ? - could do the otherwise bad `from wt_utils import *` if so, and save the reader looking at a huge imports list :grin: